auto-config.json

```diff
@@ -0,0 +1,24 @@
+{   
+    "note": "This is an instruction file used by Microsoft tools to generate Quickstart. App developers do not need to use this file.",
+    "$schema": "https://raw.githubusercontent.com/microsoft/vscode-ms-entra/main/schemas/sampleConfigV1.schema.json",
+    "_version": "1",
+    "credential": "secret",
+    "appUpdates": {
+        "web": {
+            "redirectUris": [
+                "http://localhost:5000/getAToken"
+                ]
+        }
+    },
+    "replacements": {
+        ".env.sample.entra-id":{
```

Comment:
LGTM.

The title of this PR was "add **external ID** config file", but this line uses the template for **entra-id**. Was the PR title a typo? @mwanzias 

By the way, I guess this PR will add one quickstart. Do we envision that in the future there will be 4 different `auto-config.json` in total in this repo?